xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_version.xsd

```diff
@@ -620,6 +623,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Maximum allowed Stay as Duration.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="BayGeometry" type="BayGeometryEnumeration" minOccurs="0">
```

Comment:
What does the +v1.2.2 mean?

I also notice two spaces opposed to the single space in +v1.1 below